src/corehost/cli/fxr_resolver.cpp

```diff
@@ -102,13 +102,21 @@ bool fxr_resolver::try_get_path(const pal::string_t& root_path, pal::string_t* o
             pal::get_default_installation_dir(&default_install_location);
         }
 
+		pal::string_t self_registered_config_location;
+		pal::get_dotnet_self_registered_config_location(&self_registered_config_location);
+		pal::string_t self_registered_message =
+			self_registered_config_location.length() > 0 ?
+			pal::string_t(_X(" or register the runtime location in [") + self_registered_config_location + _X("]")) :
```

Comment:
What does `register the runtime location` mean? Should this instead be:

`"or install the runtime in the globally registered location [") + self_registered_config_location + _X("]"`

?

Reply:
If there is a globally registered location already set (in registry or the config file), then that path will be listed in the first part of the error message. This part of the error message is about the registry key (or config file) itself, so the user can register the install location by setting the registry key (or writing to the config file).

The error now looks like:
```
A fatal error occurred. The required library libhostfxr.so could not be found.
If this is a self-contained application, that library should exist in [/tmp/console2/bin/Debug/netcoreapp3.0/linux-x64/publish/].
If this is a framework-dependent application, install the runtime in the global location [/usr/share/dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime in [/etc/dotnet/install_location].
```

In this case the `[/usr/share/dotnet]` is the currently active global location. So if the `/etc/dotnet/install_location` exists and points to for example `/tmp/dotnet3` then the first part of the error message would contain that path instead (it's the path where we looked for the missing 
 runtime).